### PR TITLE
[14.0][MIG] account_payment_return: Migration to 14.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
@Tecnativa
TT28727

ping @chienandalu @pedrobaeza

### CODE REVIEW

File: account_payment_return/data/ir_sequence_data.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     Copyright 2015 Sergio Teruel <sergio.teruel@tecnativa.com>
+     Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/data/ir_sequence_data.xml]:
```suggestion
<!-- Copyright 2015 Tecnativa - Pedro M. Baeza
     Copyright 2015 Tecnativa - Sergio Teruel
     Copyright 2016 Tecnativa - Carlos Dauden
```

File: account_payment_return/models/account_journal.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2017 luis M. Ontalba <luis.martinez@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/models/account_journal.py]:
```suggestion
# Copyright 2017 Tecnativa - luis M. Ontalba
```

File: account_payment_return/models/account_move.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/models/account_move.py]:
```suggestion
# Copyright 2016 Tecnativa - Carlos Dauden
```

File: account_payment_return/models/payment_return.py [hunk 4]
```diff
@@ -0,0 +1,480 @@
+# Copyright 2011-2012 7 i TRIA <http://www.7itria.cat>
+# Copyright 2011-2012 Avanzosc <http://www.avanzosc.com>
+# Copyright 2013 Pedro M. Baeza <pedro.baeza@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/models/payment_return.py]:
```suggestion
# Copyright 2013 Tecnativa - Pedro M. Baeza
```

File: account_payment_return/models/payment_return.py [hunk 5]
```diff
@@ -0,0 +1,480 @@
...
+# Copyright 2014 Markus Schneider <markus.schneider@initos.com>
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2017 Luis M. Ontalba <luis.martinez@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/models/payment_return.py]:
```suggestion
# Copyright 2016 Tecnativa - Carlos Dauden
# Copyright 2017 Tecnativa - Luis M. Ontalba
```

File: account_payment_return/models/payment_return_reason.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/models/payment_return_reason.py]:
```suggestion
# Copyright 2016 Tecnativa - Carlos Dauden
```

File: account_payment_return/security/account_payment_return_security.xml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Sergio Teruel <sergio.teruel@tecnativa.com>
+     Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/security/account_payment_return_security.xml]:
```suggestion
<!-- Copyright 2015 Tecnativa - Sergio Teruel
     Copyright 2016 Tecnativa - Carlos Dauden
```

File: account_payment_return/static/src/xml/account_payment_return.xml [hunk 8]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/static/src/xml/account_payment_return.xml]:
```suggestion
<!-- Copyright 2016 Tecnativa - Carlos Dauden
```

File: account_payment_return/tests/test_payment_return.py [hunk 9]
```diff
@@ -0,0 +1,283 @@
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2017 Luis M. Ontalba <luis.martinez@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/tests/test_payment_return.py]:
```suggestion
# Copyright 2015 Tecnativa - Pedro M. Baeza
# Copyright 2016 Tecnativa - Carlos Dauden
# Copyright 2017 Tecnativa - David Vidal
# Copyright 2017 Tecnativa - Luis M. Ontalba
```

File: account_payment_return/views/account_journal_view.xml [hunk 10]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Luis M. Ontalba <luis.martínez@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/views/account_journal_view.xml]:
```suggestion
<!-- Copyright 2017 Tecnativa - Luis M. Ontalba
```

File: account_payment_return/views/account_move_views.xml [hunk 11]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     Copyright 2015 Sergio Teruel <sergio.teruel@tecnativa.com>
+     Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/views/account_move_views.xml]:
```suggestion
<!-- Copyright 2015 Tecnativa - Pedro M. Baeza
     Copyright 2015 Tecnativa - Sergio Teruel
     Copyright 2016 Tecnativa - Carlos Dauden
```

File: account_payment_return/views/payment_return_view.xml [hunk 12]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+     Copyright 2015 Sergio Teruel <sergio.teruel@tecnativa.com>
+     Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (victoralmau) [inline account_payment_return/views/payment_return_view.xml]:
```suggestion
<!-- Copyright 2015 Tecnativa - Pedro M. Baeza
     Copyright 2015 Tecnativa - Sergio Teruel
     Copyright 2016 Tecnativa - Carlos Dauden
```

**Review by victoralmau — APPROVED** [review]

File: account_payment_return/models/payment_return.py [hunk 13]
```diff
@@ -0,0 +1,480 @@
...
+        selection=[
+            ("draft", "Draft"),
+            ("imported", "Imported"),
+            ("done", "Done"),
+            ("cancelled", "Cancelled"),
+        ],
+        string="State",
+        readonly=True,
+        default="draft",
+        track_visibility="onchange",
```
Comment (chienandalu) [inline account_payment_return/models/payment_return.py]:
```suggestion
        tracking=True,
```

**Comment by joao-p-marques** [comment]
@chienandalu @victoralmau Changes made. Thank you for the review!

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
Code review

**Comment by pedrobaeza** [comment]
@chienandalu do you confirm is OK for you?

**Review by chienandalu — APPROVED** [review]
Code and functional review :+1:

**Pull request merged**